packages/react-vapor/src/components/select/tests/MultiSelectWithPredicate.spec.tsx

```diff
@@ -150,5 +152,44 @@ describe('Select', () => {
                 expect(spy).toHaveBeenCalledWith(unselectListBoxOption(id, items[1].value));
             });
         });
+
+        describe('when predicates are processed on the server side', () => {
+            const ServerSideMultiSelectWithPredicates = _.compose(
+                withServerSideProcessing,
+                selectWithPredicate
+            )(MultiSelectWithPredicate);
+
+            const items = [{value: 'a', hidden: true}, {value: 'b', selected: true}, {value: 'c', selected: true}];
+
+            it('should not filter the items based on any predicate because it is done on the server', () => {
+                const component: ShallowWrapper<ISelectWithPredicateProps> = shallowWithStore(
+                    <ServerSideMultiSelectWithPredicates
+                        {...basicProps}
+                        items={items}
+                        predicate={basicProps.options[1].id}
+                    />,
+                    store
+                )
+                    .dive()
+                    .dive();
+                expect(component.props().items).toEqual(items);
+            });
+
+            it('should trigger the onUpdate prop when the selected predicate changes', () => {
+                const onUpdateSpy = jasmine.createSpy('onUpdate');
+                const component: ShallowWrapper<ISelectWithPredicateProps> = shallowWithStore(
+                    <ServerSideMultiSelectWithPredicates {...basicProps} onUpdate={onUpdateSpy} />,
+                    store
+                )
+                    .dive()
+                    .dive();
+
+                expect(onUpdateSpy).not.toHaveBeenCalled();
+
+                component.setProps({predicate: 'some-other-predicate'});
+
+                expect(onUpdateSpy).not.toHaveBeenCalledTimes(1);

```

Comment:
remove `.not` ?

Reply:
woops indeed and the test is probably failing then :disappointed:

Reply:
fixed